tests/compiler/continue.optimized.wat

```diff
@@ -5,26 +5,25 @@
  (export "testInherit" (func $continue/testInherit))
  (func $continue/testInherit (; 0 ;) (param $0 i32)
   (local $1 i32)
-  loop $loop|0
-   block $break|0
-    local.get $1
-    i32.const 10
-    i32.ge_s
-    br_if $break|0
-    block $continue|0
+  loop $for-loop|0
+   local.get $1
+   i32.const 10
+   i32.lt_s
+   if
+    block $for-continue|0
      local.get $0
      i32.const 0
      local.get $1
      i32.const 5
      i32.eq
      select
-     br_if $continue|0
+     br_if $for-continue|0
```

Comment:
This one is interesting in that it exits the block anyway. Wondering if Binaryen could somehow optimize all of this away.